docs/sources/clients/promtail/troubleshooting.md

```diff
@@ -28,21 +28,21 @@ Each log entry contains four fields:
 - labels
 - extracted fields
 
-Whenever any of these change, a log message will be printed:
+Enable the inspection output using the `--inspect` command-line option.<br>
+The `--inspect` option can be used in combination with `--stdin` and `--dry-run`.
```

Comment:
Please inject a blank line instead of using HTML.

```suggestion
Enable the inspection output using the `--inspect` command-line option.

The `--inspect` option can be used in combination with `--stdin` and `--dry-run`.
```